lib/internal/modules/esm/loader.js

```diff
@@ -393,7 +402,7 @@ class ModuleLoader {
       parentURL,
     };
 
-    return defaultResolve(originalSpecifier, context);
+    return defaultResolve(originalSpecifier, context, this.#importMap);
```

Comment:
Should importmap be added to `context`? 🤔 (especifally because the 3rd argument of all hook chains is `next`)